styles/styles.css

```diff
@@ -0,0 +1,132 @@
+
+.container{
+display: grid;
+grid-template: 1fr 2fr 2fr .5fr 2fr 2fr 3.5fr 2fr auto / 2fr 1fr 1fr;
+grid-template-areas: 
```

Comment:
great use of named areas here. this can be super helpful when wrapping your head around complex grid layouts.